crm_action/models/crm_lead.py

```diff
@@ -1,4 +1,4 @@
-# -*- encoding: utf-8 -*-
+# -*- coding: utf-8 -*-
 ##############################################################################
 #
 #    OpenERP, Open Source Management Solution
```

Comment:
Please update these awful license headers and put the shorter ones if you don't mind.